lib/spree/shipment_notice.rb

```diff
@@ -10,13 +10,28 @@ def initialize(params)
     end
 
     def apply
-      locate ? update : not_found
+      return not_found unless locate
+      paid ? update : not_paid
     rescue => e
       handle_error(e)
     end
 
     private
 
+    def paid
+      order = shipment.order
+      return true if order.paid?
+
+      # We try to capture payments if flag is set
+      paid = if Spree::Config.shipstation_capture_at_notification
```

Comment:
Can we take out the `paid` variable here? I don't think we really need it, also if the `capture_at_notification` setting is set to false is this always going to set an error?